honeybadger/version.py

```diff
@@ -1 +1 @@
-__version__ = '0.0.6'
+__version__ = '0.1.0'
```

Comment:
Can you remove the version bump from this PR? We'll do the version bump separately (after merging) so that we can cleanly tag the release.